src/config/README.md

```diff
@@ -1,9 +1,13 @@
 # About this folder
 
 This folder holds configuration files for different environments.
+
 You can use it to provide your app with different settings based on the 
 current environment, e.g. to configure different API base urls depending on 
 whether your setup runs in dev mode or is built for distribution.
+
+***Note that there is a Webpack alias defined in /cfg/base.js which redirects `'config'` to `'${projectRoot}/config/$env'`***
```

Comment:
🤔 Can you specify _why_ the redirection is interesting or surprising? What in the development process made you decide to update the `README.md`?

Reply:
Good question. The redirection itself is neither interesting or surprising. What was not immediately clear to me is where the redirection alias was defined (note that there are two places for configuration ./cfg/ and ./src/config/).

Also, note that this is not the main README file, but a config specific README file. So, it made sense to me to put the note here to help people understand the config pipeline a bit more.